### DISCUSSION TITLE
Proporties - Sound Power ESS

### DISCUSSION THREAD

**Discussion by JessicaFerrari89** [Ifsttar/I-Simpa#380] [Q&A] [answered]
Hi everyone, for my simulation I need to send an exponential sine sweep ESS from the sound sources but if I access the properties - sound power panel I don't find the possibility of using a sine sweep in the list and it doesn't even let me modify the table data manually. Is there perhaps a way to modify this part with python in order to manually decide the dB values for each frequency band? Have you already done it and tried it?
A thousand thanks.

**Comment by nicolas-f** [comment, marked as answer]
Hi,

You can define your own spectrum in the I-Simpa database:

https://i-simpa-wiki.readthedocs.io/fr/latest/project_database.html#spectrum

However as SPPS is modelling an impulse source, you can not emit a sine sweep.

best regards,

**Reply by JessicaFerrari89** [reply]
Ok, thank you for the explanation.
